tdrs-backend/tdpservice/search_indexes/parsers/models.py

```diff
@@ -0,0 +1,48 @@
+"""Models representing parser error."""
+
+from django.db import models
+from django.contrib.contenttypes.fields import GenericForeignKey
+from django.contrib.contenttypes.models import ContentType
+
+class ParserError(models.Model):
+    """Model representing a parser error."""
+
+    class Meta:
+        """Meta for ParserError."""
+
+        db_table = "parser_error"
+
+    id = models.AutoField(primary_key=True)
+    file = models.ForeignKey(
+        "data_files.DataFile",
+        on_delete=models.CASCADE,
+        related_name="parser_errors",
+        null=True,
+    )
+    row_number = models.IntegerField(null=False)
+    column_number = models.IntegerField(null=False)
+    item_number = models.IntegerField(null=False)
+    field_name = models.TextField(null=False, max_length=128)
+    category = models.IntegerField(null=False, default=1)
+
```

Comment:
After conversation with UX, would like to include case number and english representation of month (e.g. January, instead of '1').